misc/package/Data Files/MWSE/core/mcm/components/settings/Setting.lua

```diff
@@ -23,8 +24,7 @@ function Setting:new(data)
 	if data and data.variable then
 		-- create setting variable
 		t.variable.defaultSetting = t.variable.defaultSetting or t.defaultSetting
-		local typePath = ("mcm.variables." .. t.variable.class)
-		t.variable = require(typePath):new(t.variable)
+		t.variable = fileUtils.getVariableClass(t.variable.class):new(t.variable)
```

Comment:
This change doesn't have much impact on performance, but it maintains consistency with the other changes. It also helps to make sure `fileUtils` is solely responsible for all logic that associates a Lua file to a given `className`.